### PR TITLE
DOC: Small doc. fix in UCP

### PR DESCRIPTION
**Pull request by tonycurtis**
## What
* remove "an" in ucp.h:ucp_context_query()
* update English in ucp.h:ucp_context_print_info().
    * also can/should we make the context parameter "const" since it's an inspection routine?

## Why ?
Fixing English.

## How ?
N/A

### CODE REVIEW

**Comment by shamisp** [comment]
Do we have const in any other API UCP interfaces ?

**Comment by tonycurtis** [comment]
> On Mar 4, 2019, at 12:13 PM, Pavel Shamis (Pasha) <notifications@github.com> wrote:
> 
> Do we have const in any other API UCP interfaces ?
> 

Yes, quite a few (e.g. send/put, rkey_unpack).
tony

**Comment by shamisp** [comment]
👍

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9125/ for details (Mellanox internal link).

File: src/ucp/api/ucp.h [hunk 1]
```diff
@@ -1189,14 +1189,14 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
...
  *
- * This routine prints information about the context configuration, including
+ * This routine prints information about the context configuration: including
  * memory domains, transport resources, and other useful information associated
  * with the context.
  *
- * @param [in] context      Context object whose configuration to print.
- * @param [in] stream       Output stream to print the information to.
+ * @param [in] context      Print this context object's configuration.
+ * @param [in] stream       Output stream to which to print the information.
```
Comment (bbenton) [inline src/ucp/api/ucp.h]: "Output stream **_on_** which to print..." sounds a bit better to me.

Comment (tonycurtis) [reply src/ucp/api/ucp.h]: Yeah, agreed.

**Comment by bbenton** [comment]
:+1:

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9130/ for details (Mellanox internal link).

**Pull request merged**